CHANGELOG.md

```diff
@@ -4,21 +4,45 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 2.5.1 (2021-12-01)
 
-**Features**
-* #3768 Allow ALTER TABLE ADD COLUMN with DEFAULT on compressed hypertable
-* #3769 Allow ALTER TABLE DROP COLUMN on compressed hypertable
+This release contains bug fixes since the 2.5.0 release.
+We deem it medium priority to upgrade.
 
 **Bugfixes**
+* #3706 Test enabling dist compression within a procedure
+* #3734 Rework distributed DDL processing logic
+* #3737 Fix flaky pg_dump
 * #3739 Fix compression policy on tables using INTEGER
+* #3752 Add PG14 to package tests
+* #3756 Refactor function in dist_commands.c
+* #3757 Add Postgres 14 prerequisites for compilation
+* #3759 Move ChunkAppend and AsyncAppend code into nodes directory
+* #3762 Refactor ts_hypertable_insert_path_create
+* #3763 Fix typo in error message
```

Comment:
Same here. Not need since they are internal changes.
```suggestion
* #3762 Refactor ts_hypertable_insert_path_create
* #3763 Fix typo in error message
```